### ISSUE TITLE
Do you have any plans to complete another plugin, Obsidian Tasks Calendar, for 702573N

### ISSUE THREAD

**Issue by fireflysss** [Leonezz/obsidian-tasks-calendar-wrapper#73]
Do you have a plan to https://github.com/702573N/Obsidian-Tasks-Calendar ，This is also made into a plugin, which feels very beautiful. If it can be linked with the tasks calendar wrapper, it would be perfect

**Comment by Leonezz** [comment]
Yes there is a plan for the calendar view, but I am afraid it will take a long time since I am currently busy with something else.

**Comment by RaviOnline** [comment]
This plugin should have been named "timeline" wrapper then. It's misleading to call it a "calendar" wrapper when it doesn't provide any calendar view yet and there are no immediate plans to implement the same.

Don't get me wrong, your work is certainly worth appreciation. But, the way plugin search works, your plugin comes up when looking for a task calendar and it's disappointing to find out after installation that it doesn't do what it claims to do.

**Comment by Leonezz** [comment]
> This plugin should have been named "timeline" wrapper then. It's misleading to call it a "calendar" wrapper when it doesn't provide any calendar view yet and there are no immediate plans to implement the same.
> 
> Don't get me wrong, your work is certainly worth appreciation. But, the way plugin search works, your plugin comes up when looking for a task calendar and it's disappointing to find out after installation that it doesn't do what it claims to do.

You are right, I am not entirely sure if the calendar and timeline view should be put in one plugin, I will try to change the plugin name and maybe also the repo name if I decide to make another plugin for calendar view.

**Comment by fireflysss** [comment]
I'm glad you appreciate the response! If this calendar plugin could implement Gantt charts, display task progress bars, enable dragging to change timelines, and allow dragging progress bars to adjust time spans – similar to Google Calendar or Apple Calendar functionality – that would be fantastic! You're already great, of course!
